tests/DbaaS/pages/dbaasActionsPage.js

```diff
@@ -24,10 +24,10 @@ module.exports = {
   },
 
   async createClusterBasicOptions(k8sClusterName, dbClusterName, dbType, dbVersion) {
-    I.waitForElement(dbaasPage.tabs.dbClusterTab.dbClusterAddButtonTop, 30);
-    I.waitForInvisible(dbaasPage.tabs.kubernetesClusterTab.disabledAddButton, 30);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.dbClusterAddButtonTop, 3);
```

Comment:
Timeout 30 is too long, I think 3sec should be enough for most cases.
It is often race condition issue (due to parallel execution), not a slow execution.  It is better to fail faster and retry test